packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
```

Comment:
Is there a reason we aren't using `Convert-Path $PSScriptRoot`? It's more idiomatic PowerShell and does the same thing.